x-pack/plugins/cases/public/components/connectors/case/alert_fields.tsx

```diff
@@ -89,9 +89,17 @@ const CaseParamsFields: React.FunctionComponent<ActionParamsProps<CaseActionPara
     actionParams.subAction,
   ]);
 
+  /**
+   * TODO: When the case connector is enabled we need to figure out
```

Comment:
Just curious. What's the issue here?

Reply:
Case as a connector isn't supported yet (the sub cases feature). I think this TODO is stating that we don't have a configurable way to pass `owner` to this component yet. Below we're hardcoding it to the security solution but if we support case as a connector in other plugins (observability) we'll need to allow the `owner` to be passed in like the other case components.